sections/icon-message.liquid

```diff
@@ -0,0 +1,4 @@
+<svg width="18" height="16" viewBox="0 0 18 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
use snippets for icons instead of sections. And render them inside sections where needed.